src/main/java/jp/kusumotolab/kgenprog/KGenProgMain.java

```diff
@@ -281,12 +287,37 @@ private double getNormalizedFitnessValue(final Variant variant) {
         .getNormalizedValue();
   }
 
-  private void logGAStopped(final OrdinalNumber generation) {
+  private DecimalFormat createDecimalFormat() {
+    return new DecimalFormat("#.###");
+  }
+
+  private void logGAStopped(final OrdinalNumber generation, final String time,
+      final ExitStatus exitStatus) {
     final StringBuilder sb = new StringBuilder();
     sb//
-        .append("GA stopped at the era of ")
-        .append(generation.toString())
-        .append(" generation.");
+        .append("Overall")
```

Comment:
MTGで話してたやつ
Summaryの方がいいかも

```suggestion
        .append("Summary")
```